README.md

```diff
@@ -169,6 +169,14 @@ This initialises a new directory and Git repository.
   a message queue is employed between the source topic and the Lambda function,
   and unprocessed events are sent to a dead-letter queue for manual triage.
 
+- `lambda-sqs-worker-cdk`
+
+  An asynchronous [worker] built on [AWS Lambda] and deployed with [AWS CDK].
```

Comment:
This doesn't seem in line with the Technology Strategy: https://tech-strategy.ssod.skinfra.xyz/docs/v1/technology.html#lambda

I don't mind merging this, but can we kick off internal discussions over the merit of the CDK as a Lambda deployment tool?

Reply:
Possibly a misunderstanding on my part:
https://tech-strategy.ssod.skinfra.xyz/docs/v1/components.html#infrastructure

I read it to include Lambda workers. We're already using CDK to deploy workers in practice (among other things).

Reply:
I think @72636c is referring to:
> Lambda functions SHOULD be deployed using [Serverless](https://serverless.com/)

Allowing CDK usage for infrastructure wouldn’t override the above guidance